### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by rrodgeur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
 # control_library
-control and dsp algorithms for owntech converters
+
+Control and dsp algorithms for owntech converters
+
+This library has been integrated and tested on [TWIST](https://github.com/owntech-foundation/TWIST) converters using [SPIN](https://github.com/owntech-foundation/SPIN) controller.
+
+The integration is based on the dedicated [OwnTech Power API](https://github.com/owntech-foundation/Core).
+
+__Remarks__: The library is currently not optimized to be (we hope) easier to read.
```
Comment (jalinei) [inline README.md:9]: Not english

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+
+## Features
+
+The library provides the following functionalities mainly through C++ objects :
+ * `Pid()`: standard form of the PID regulator.
```
Comment (jalinei) [inline README.md:14]: Standard

File: README.md [hunk 3]
```diff
@@ -1,2 +1,60 @@
...
+  * K. Astrom et T. Hagglund, Advanced PID Control.
+* PR:
+  * A. Gï, « Digital Resonant Current Controllers For Voltage Source Converters ». [En ligne]. Disponible sur: http://agyepes.webs.uvigo.es/files/Thesis.pdf
+* RST:
+  * I. D. Landau, C. Cyrot, A. Voda, et D. Rey, « Robust digital control of flexible structures using the combined pole placement/sensitivity function shaping method », in Proceedings of 1994 American Control Conference - ACC ’94, Baltimore, MD, USA: IEEE, 1994, p. 283‑288. doi: 10.1109/ACC.1994.751743.
+  * I. D. Landau et G. Zito, « Digital Control Systems - New edition (I. D. Landau &amp; G. Zito) », 2020, doi: 10.13140/RG.2.2.19321.49764.
+* PLL:
+  * F. D. Freijedo, J. Doval-Gandoy, O. Lopez, et J. Cabaleiro, « Robust phase locked loops optimized for DSP implementation in power quality applications », in 2008 34th Annual Conference of IEEE Industrial Electronics, Orlando, FL: IEEE, nov. 2008, p. 3052‑3057. doi: 10.1109/IECON.2008.4758447.
+ 
+## Thanks
```
Comment (jalinei) [inline README.md:52]: Credits ?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,60 @@
...
+
+Thanks to the OwnTech contributors:
+
+* [Luiz](https://github.com/luizvilla)
```
Comment (jalinei) [inline README.md:56]: Put yourself first as you are the main author and contributor of this work ?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,60 @@
...
+
+To use the library, you need to add the line above in the `platformio.ini` file.
+
+```ini
+lib_deps=
+    control_lib = https://github.com/owntech-foundation/control_library.git
+```
+
+You can find various examples in the [OwnTech examples library](https://github.com/owntech-foundation/examples) 
+
```
Comment (jalinei) [inline README.md:37]: Licence section missing

Comment (rrodgeur) [reply README.md:37]: There's a tab for that in Github, do I have to add it anyway  ? or put a link ?
image

**Pull request merged**